src/components/appheadertop.main.jsx

```diff
@@ -28,15 +28,18 @@ import AppHeaderLocaleMain from './appheaderlocale.main';
 
 class AppHeaderTop extends React.Component {
   static propTypes = {
-    isMobileView: PropTypes.bool.isRequired,
+    isMobileView: PropTypes.bool,
   }
 
+  static defaultProps = {
+    isMobileView: false,
+  }
 
   render() {
     const { isMobileView } = this.props;
 
     return [
-      <div className={`top-header ${isMobileView ? 'mobile-view' : ''}`}>
+      <div key={`AppHeaderTop_${Math.random().toString(36).substr(2, 9)}`} className={`top-header ${isMobileView ? 'mobile-view' : ''}`}>
```

Comment:
it is better to move as util function:
getUniqueKey(key) {
 return `${key}_${Math.random().toString(36).substr(2, 9)}`
}

Reply:
Thanks @skarpyak! I changed this to a static key since there wasn't a need for it to be unique on the page.